src/App.test.js

```diff
@@ -0,0 +1,9 @@
+import React from "react";
+import ReactDOM from "react-dom";
+import App from "./App";
+
+it("renders without crashing", () => {
```

Comment:
You should encapsulate the `it` statement inside of a `describe` block. It helps to isolate code, makes finding tests easier, and helps you to debug.

Usually I break up my unit tests by creating a `describe` block for every class|function. In this case you only have a function that renders a React component, so I would do:

```
describe('App', () => {
  it("renders without crashing", () => {
    const div = document.createElement("div");
    ReactDOM.render(<App />, div);
    ReactDOM.unmountComponentAtNode(div);
  });
});
```